src/modules/campaign-monitor/services/campaign-monitor.service.spec.ts

```diff
@@ -193,33 +194,39 @@ describe('CampaignMonitorService', () => {
 			);
 			sendTransactionalMailSpy.mockResolvedValueOnce(undefined);
 
-			await campaignMonitorService.sendForVisit({
-				to: [{ id: visit.visitorId, email: null, language: Locale.Nl }],
-				template: Template.VISIT_APPROVED,
-				visitRequest: visit,
-			});
-			expect(sendTransactionalMailSpy).toBeCalledWith({
-				template: Template.VISIT_APPROVED,
-				data: {
-					to: ['MEEMOO_MAINTAINER_MISSING_EMAIL_FALLBACK'],
-					consentToTrack: 'unchanged',
+			await campaignMonitorService.sendForVisit(
+				{
+					to: [{ id: visit.visitorId, email: null, language: Locale.Nl }],
+					template: Template.VISIT_APPROVED,
+					visitRequest: visit,
+				},
+				'nl'
```

Comment:
why do you pass language here when it is already passed for each receiver above?
```
to: [{ id: visit.visitorId, email: null, language: Locale.Nl }],
```